commit 8b6d548dd352e6069f2ec5063edbe105edf17869 (refs/changes/23/1323/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-01-24T21:08:06+01:00 (2 years, 8 months ago)
    
    joyent/node-cueball#76 EventEmitter listeners limit should be increased
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/lib/resolver.js b/lib/resolver.js
index 3ee2fb3..2f0502e 100644
--- a/lib/resolver.js
+++ b/lib/resolver.js
@@ -3,7 +3,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  *
- * Copyright (c) 2016, Joyent, Inc.
+ * Copyright (c) 2017, Joyent, Inc.
  */
 
 module.exports = {
@@ -262,6 +262,7 @@ function CueBallDNSResolver(options) {
 		this.r_service = '_dns._udp';
 		this.r_defport = 53;
 		this.r_maxres = 10;
+		this.setMaxListeners(500);
 	}
 
 	mod_assert.optionalObject(options.log, 'options.log');
